[F10-2] Health Booklet

NichPang1 opened the pull request:
@C-E-OOI  @OngJiaJian  @keithtqs

fanyiii commented:
For consistency, the travis badge in README should also direct to the one on travis-ci.org. You can aid the description of feature implementations with diagrams that we have learned in the course. Also, remember to be consistent in the naming and avoid using AddresBook when you are building Health Booklet. You should also put in your own architecture diagram, which shows high-level info about major components in your app.

NichPang1 committed:
Merge pull request #114 from NichPang1/master

Edited ClearCommand success message

NichPang1 committed:
Update src/seedu/addressbook/common/Messages.java

NichPang1 committed:
Merge pull request #115 from NichPang1/master

Edited Person not in addressbook error message

NichPang1 committed:
Fixed bug where undo/redo for multiple clear commands in the same session does not work properly

NichPang1 committed:
Update src/seedu/addressbook/commands/ClearCommand.java

NichPang1 committed:
Merge pull request #116 from NichPang1/master

 Fixed bug where undo/redo for multiple clear commands in the same session does not work properly

ojayjae committed:
Update config.json

ojayjae committed:
added Access Level field in login class

C-E-OOI committed:
sucessful list apoointment dates

C-E-OOI committed:
better display messages for edit appointment

C-E-OOI committed:
User friendly messages

ojayjae committed:
Added AccessLevel field to credentials and updated relevant methods

ojayjae committed:
In login UI,
enter button will activate login
empty username and passwords will be rejected

ojayjae committed:
Added NRIC field + edit tests to include NRIC

C-E-OOI committed:
Merge pull request #118 from OngJiaJian/master

Added AccessLevel and Updated UI

ojayjae committed:
Change GUI to include logged in user at bottom or UI

ojayjae committed:
GUI shows logged in account access level as well

ojayjae committed:
Change accesslevel field to int from String

ojayjae committed:
implemented accesslevel on Address field
if no access level, address field will be HIDDEN

ojayjae committed:
Edited UserGuide.adoc to v.13

ojayjae committed:
AccessLevel less than or equals to now
Changepassword command will remove spaces

C-E-OOI committed:
Merge pull request #121 from OngJiaJian/master

Implemented Accesslevel on Address field

C-E-OOI committed:
edit-appointment modification

C-E-OOI committed:
add command for edit schedule added but list command is removed as it is unable to be updated immediately. The implementation for update of scheduel to immunatable list is still under construction due to a bug which causes null pointer whenever it is runned.(error causing line is currently commented away

C-E-OOI committed:
delete appointmentd for edit-appointments is up

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into scheduleEdit

# Conflicts:
#	src/seedu/addressbook/login/loginstorage.txt

C-E-OOI committed:
resolved  bug that caused test failure and further edited it

NichPang1 committed:
Merge pull request #127 from C-E-OOI/scheduleEdit

Resolved bug that causes test failure previously

keithtqs committed:
PPP

C-E-OOI committed:
Merge pull request #128 from keithtqs/master

PPP

keithtqs committed:
change to adoc format

C-E-OOI committed:
Merge pull request #129 from keithtqs/master

change to adoc format

NichPang1 committed:
Some refractoring

C-E-OOI committed:
edit-appoinment feature is now fully functional! messages as also updated

NichPang1 committed:
Wrote more tests for undo and link

NichPang1 committed:
Wrote more tests on link and undo

NichPang1 committed:
wrote more tests undo

NichPang1 committed:
Update test/java/seedu/addressbook/logic/LogicTest.java

C-E-OOI committed:
edit the history and enhanced the messages

C-E-OOI committed:
Merge pull request #130 from NichPang1/master

Some refractoring and tests

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into scheduleEdit

# Conflicts:
#	src/seedu/addressbook/commands/ListAppointment.java

NichPang1 committed:
Merge pull request #133 from C-E-OOI/scheduleEdit

edit-appointment is now fully functional

C-E-OOI committed:
cleaned up the code - deleting unneeded comment made and getting rid of unused imports

C-E-OOI committed:
message for add appointment and delete appointment has enhanced message and cleaned

C-E-OOI committed:
clean

ojayjae committed:
Added new Login when adding a Doctor as Administrator
Username is NRIC and Password is "StandardPass123!"

ojayjae committed:
Merge pull request #134 from C-E-OOI/scheduleEdit

Clean up and enhance message for addappoinment and deleteappointment

ojayjae committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

# Conflicts:
#	src/seedu/addressbook/login/loginstorage.txt
 resolved

NichPang1 committed:
Unlink command

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

NichPang1 committed:
Merge pull request #135 from OngJiaJian/master

new Account will be created when adding Doctor when logged in as Admin

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

C-E-OOI committed:
limit the date range for schedule

C-E-OOI committed:
display minimal message for list

ojayjae committed:
Change Duplicate Person to be checked via only the NRIC field currently

C-E-OOI committed:
command word no longer case sensitive

ojayjae committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

ojayjae committed:
Edit test to to cater for Duplicate Person Exception

C-E-OOI committed:
clean

C-E-OOI committed:
Merge pull request #137 from OngJiaJian/master

Updated how duplicate person works

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into scheduleEdit

ojayjae committed:
Merge pull request #139 from C-E-OOI/scheduleEdit

Minimal message for list, command word no longer case sensitive, clean

NichPang1 committed:
Implemented undo for CHange password and hardcoded some sample data

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

ojayjae committed:
Added loginUI and changed mainUI in UserGuide

ojayjae committed:
Merge pull request #140 from NichPang1/master

Created UnLinkCommand, implemented undo/redo for change password, hardcoded sample data on first run

ojayjae committed:
Copied Diagrams from AB4 to AB3

ojayjae committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

ojayjae committed:
Allocated slides to prevent conflicts

ojayjae committed:
Revert "Allocated slides to prevent conflicts"

This reverts commit 1acbd977d5c309246cf96f2936ca85ce1673b87b.

ojayjae committed:
Merge pull request #143 from OngJiaJian/master

Updated USerguide

NichPang1 committed:
COmments for code

NichPang1 committed:
Revert "COmments for code"

This reverts commit 8dea8b7688fac93548569e1a68013cdb6d44d3f3.

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

NichPang1 committed:
Comments on CommandClass

NichPang1 committed:
Comments for UndoAbleCommand

NichPang1 committed:
Comments for undo/redo

NichPang1 committed:
Revert "Comments for undo/redo"

This reverts commit 4ed39fb85d25de80caf198d41e987508727ca018.

NichPang1 committed:
Revert "Comments for UndoAbleCommand"

This reverts commit 7df4fa5e0cf3bda2e7fd1e5b0ed4278ea32f7ead.

NichPang1 committed:
Revert "Comments on CommandClass"

This reverts commit 4887766c83959d18b73656322b458ad7473a709c.

C-E-OOI committed:
structure of our UG and help list

keithtqs committed:
V1.4 Updates

keithtqs committed:
V1.4 Updates

keithtqs committed:
V1.4 Updates

keithtqs committed:
V1.4 Updates

keithtqs committed:
Merge pull request #144 from C-E-OOI/documentationAndTest

Structure of our UG and help list

keithtqs committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

# Conflicts:
#	src/seedu/addressbook/commands/HelpCommand.java

C-E-OOI committed:
edited user guide

keithtqs committed:
Merge pull request #147 from C-E-OOI/documentationAndTest

edited user guide (short pr to minimise conflict)

keithtqs committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

keithtqs committed:
V1.4 Updates

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

NichPang1 committed:
Revert "Revert "Comments on CommandClass""

This reverts commit 49b26dfc8a141b1b2539b507eae97fa74b100282.

NichPang1 committed:
Revert "Revert "Comments for UndoAbleCommand""

This reverts commit 315eebad0cd28597b4f2ad073bcdd2cf96fb3e22.

NichPang1 committed:
Revert "Revert "Comments for undo/redo""

This reverts commit e1ed1594d2536e80b7b03ed25c106d66ba5395c6.

C-E-OOI committed:
updated diagrams

NichPang1 committed:
Comments for command stack and some methods in addressbook

NichPang1 committed:
Comments for commandhistory

NichPang1 committed:
Comments for storagefile

NichPang1 committed:
Comments for adapted associate

NichPang1 committed:
Update src/seedu/addressbook/data/person/Associated.java

NichPang1 committed:
No more private for tittle

NichPang1 committed:
Revert "No more private for tittle"

This reverts commit 9df51c69afd0844f1280801553349f86de0ccd6e.

NichPang1 committed:
Revert "Revert "No more private for tittle""

This reverts commit 8d49a5a374d9e3101df0e5aa8d9f717f15a8bdc3.

NichPang1 committed:
Merge pull request #148 from keithtqs/master

V1.4

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

NichPang1 committed:
Update src/seedu/addressbook/commands/AddCommand.java

NichPang1 committed:
Updated add command usage for creating doctor/patient

keithtqs committed:
Merge pull request #149 from NichPang1/master

Added comments to code and removed private field for tittle

keithtqs committed:
Update README

keithtqs committed:
Added Coveralls

keithtqs committed:
Clean up code for codacy

NichPang1 committed:
Comments and stuff

NichPang1 committed:
Added adoc

NichPang1 committed:
Tests for unlink

NichPang1 committed:
Make the generatepersonwithname take in a seed so NRIC wont RNG and fail test

NichPang1 committed:
Codacy fix

NichPang1 committed:
Revert "Codacy fix"

This reverts commit 33c6b54768cd7ab9c6c408510d7b043184c2bd4b.

C-E-OOI committed:
schedule now includes 24 hour time and is in the format of DD-MM-YYYY-HH:MM

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

# Conflicts:
#	test/java/seedu/addressbook/logic/LogicTest.java
#	test/java/seedu/addressbook/storage/StorageFileTest.java

C-E-OOI committed:
update ug

C-E-OOI committed:
Merge pull request #151 from NichPang1/master

Added some comments in code, fixed the random test failures, added in a few tests and my adoc

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

NichPang1 committed:
Logic class diagram and undo sequence diagram

C-E-OOI committed:
ug informs user about title, appoinment dates and tags

NichPang1 committed:
Slightly restructured Undo/Redo/CommandStack

NichPang1 committed:
Update docs/images/UndoRedoSequenceDiagram.png

NichPang1 committed:
Update src/seedu/addressbook/data/AddressBook.java

NichPang1 committed:
Merge pull request #153 from NichPang1/master

Created sequence diagram for undo and Logic class diagram. Minor tweaks to code to match diagram

NichPang1 committed:
Merge pull request #150 from keithtqs/master

Update ReadMe and build

NichPang1 committed:
Some additions to DG and replaced all Health Booklet with MediBook

NichPang1 committed:
Merge pull request #152 from C-E-OOI/documentationAndTest

The input for appointment now takes into account time and is in the format of DD-MM-YYYY-HH:MM

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

NichPang1 committed:
Updated implementation of State Control in DG

NichPang1 committed:
Throw in the logic component diagram

keithtqs committed:
UGDG Updates and Help Command Ordering for clarity

keithtqs committed:
Merge pull request #154 from NichPang1/master

Updated implementation for DG

keithtqs committed:
Resolve merge conflict and additional language updates to DG

ojayjae committed:
Merge pull request #155 from keithtqs/master

UGDG Updates

ojayjae committed:
edited Diagrams for UI and Login class diagrams
remove unneccessary lines in login class

ojayjae committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

ojayjae committed:
Updated Class Diagrams is DeveloperGuide

ojayjae committed:
added PPP into project

ojayjae committed:
Updated Ui.png

NichPang1 committed:
Implemented undo/redo for add and delete (appointment)

NichPang1 committed:
Merge pull request #156 from OngJiaJian/master

Updated DeveloperGuide and PPP

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

NichPang1 committed:
updated diagrams to match

NichPang1 committed:
cleaned up old code

C-E-OOI committed:
written logic test case for edit-appointment

C-E-OOI committed:
Merge pull request #157 from NichPang1/master

Implemented undo/redo for add and delete (appointment)

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

keithtqs committed:
Merge pull request #159 from C-E-OOI/documentationAndTest

Written logic test for edit-appointment

C-E-OOI committed:
clean

keithtqs committed:
Update UGDG

keithtqs committed:
Update sort command and added new sort by title

keithtqs committed:
Updates to files related to the sort

NichPang1 committed:
Merge pull request #160 from keithtqs/master

Various updates detailed below

NichPang1 committed:
Updated UG/DG/help command

NichPang1 committed:
More on DG

NichPang1 committed:
minor change to message

NichPang1 committed:
clean up commented out code

C-E-OOI committed:
Merge pull request #161 from NichPang1/master

Updated UG/DG/help command

NichPang1 committed:
More update on DG

C-E-OOI committed:
updated ppp

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

ojayjae committed:
loginstorage.txt will be created in same directory of jar file

ojayjae committed:
standardised loginstorage

ojayjae committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

ojayjae committed:
updated ChangePasswordCommand messages

ojayjae committed:
Merge pull request #162 from OngJiaJian/master

Loginstorage now created if same directory as jar

ojayjae committed:
changed command word for ChangePasswordCommand to "changepassword"

ojayjae committed:
updated UserGuide to include usage for ChangePasswordCommand

ojayjae committed:
added missing command word in UserGuide

ojayjae committed:
Unable to change Admin password now
Edited HIDDEN to ***HIDDEN***

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

NichPang1 committed:
Update docs/team/nichpang1.adoc

NichPang1 committed:
Link to code

NichPang1 committed:
Merge pull request #164 from NichPang1/master

Updates to documentation

NichPang1 committed:
Adjustments to UG/DG to attempt to cut down

NichPang1 committed:
Merge pull request #163 from OngJiaJian/master

Prevent admin password from being changed and updated UserGuide

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

C-E-OOI committed:
documentations

NichPang1 committed:
Update docs/team/nichpang1.adoc

NichPang1 committed:
cut down other projects

NichPang1 committed:
Update docs/team/nichpang1.adoc

NichPang1 committed:
Update docs/team/nichpang1.adoc

NichPang1 committed:
Update docs/team/nichpang1.adoc

ojayjae committed:
Will create loginstorage.txt with initialised values

ojayjae committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

ojayjae committed:
Merge pull request #165 from NichPang1/master

Attempt to cut down ppp length

NichPang1 committed:
Update README.adoc

NichPang1 committed:
Revert "Update README.adoc"

This reverts commit a8df8c33e878d84f69b1b882d501cf4a07f58fcd.

NichPang1 committed:
test

NichPang1 committed:
test

NichPang1 committed:
Link portfolio on readme

ojayjae committed:
removed commented codes

NichPang1 committed:
Update docs/team/nichpang1.adoc

ojayjae committed:
removed test

C-E-OOI committed:
updated DG

ojayjae committed:
Merge pull request #167 from NichPang1/master

Linked our portfolios on the readme

NichPang1 committed:
Merge pull request #166 from OngJiaJian/master

Initialise loginstorage.txt and updated dev guide

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

# Conflicts:
#	docs/UserGuide.adoc

NichPang1 committed:
Manual testing section

NichPang1 committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

# Conflicts:
#	docs/DeveloperGuide.adoc

ojayjae committed:
tagged userguide contirbutions in PPP

ojayjae committed:
updated PPP to include contributions to dev guide

ojayjae committed:
Merge pull request #168 from C-E-OOI/documentationAndTest

Documentation

ojayjae committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3

C-E-OOI committed:
Merge pull request #169 from OngJiaJian/master

Updated documentation

ojayjae committed:
Deleted comemnts and unused code

NichPang1 committed:
Updated diagrams

NichPang1 committed:
Update diagrams

ojayjae committed:
updated documents

ojayjae committed:
remove a test case

NichPang1 committed:
added in one use case

C-E-OOI committed:
documnetations

keithtqs committed:
Written new tests, PPP updates, UGDG updates

NichPang1 committed:
Merge pull request #170 from OngJiaJian/master

Deleted comemnts and unused code

ojayjae committed:
Merge pull request #171 from NichPang1/master

Updated diagrams

keithtqs committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

# Conflicts:
#	docs/DeveloperGuide.adoc

keithtqs committed:
Merge branch 'master' of https://github.com/CS2113-AY1819S1-F10-2/main

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

# Conflicts:
#	docs/DeveloperGuide.adoc

C-E-OOI committed:
Merge pull request #172 from keithtqs/master

Various updates listed below

C-E-OOI committed:
Merge branch 'master' of https://github.com/CS2113-AY1819-F10-2/addressbook-level3 into documentationAndTest

NichPang1 committed:
Update docs/team/nichpang1.adoc

ojayjae committed:
updated ppp

C-E-OOI committed:
Merge pull request #173 from C-E-OOI/documentationAndTest

Documentation

NichPang1 committed:
Merge pull request #174 from NichPang1/master

Update docs/team/nichpang1.adoc

ojayjae committed:
update PPP

C-E-OOI committed:
Merge pull request #175 from OngJiaJian/master

updated ppp

ojayjae committed:
added author for reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
{
      "githubId": "OrangeJJ",
      "displayName": "ONG...IAN",
      "authorNames": ["OrangeJJ"],
      "authorNames": ["OngJiaJian"],
      "authorNames": ["ongjiajian"]
    },

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
reposense

ojayjae committed:
trying for keith

ojayjae committed:
Merge pull request #177 from OngJiaJian/master

Fixed reposense for OngJiaJian and Keith Tan

ojayjae committed:
conslolidate same authors

ojayjae committed:
Merge pull request #178 from OngJiaJian/master

Fixed Reposense
